### PR TITLE
[4.0]namespace administrator components com_installer

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_installer/Model/ManageModel.php [hunk 1]
```diff
@@ -216,7 +220,7 @@ public function remove($eid = array())
 		}
 
 		// Get an installer object for the extension type
-		$installer = \JInstaller::getInstance();
+		$installer = Installer::getInstance();
 		$row = new \Joomla\CMS\Table\Extension($this->getDbo());
```
Comment (Quy) [inline administrator/components/com_installer/Model/ManageModel.php:224]: Update

File: administrator/components/com_installer/Model/UpdateModel.php [hunk 2]
```diff
@@ -16,6 +16,14 @@
 use Joomla\CMS\Plugin\PluginHelper;
 use Joomla\Utilities\ArrayHelper;
 use Joomla\CMS\MVC\Factory\MVCFactoryInterface;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_installer/Model/UpdateModel.php]: Spelling `Fileystem`

File: administrator/components/com_installer/Model/InstallModel.php [hunk 3]
```diff
@@ -12,6 +12,15 @@
 
 use Joomla\CMS\MVC\Model\BaseDatabaseModel;
 use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Client\ClientHelper;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_installer/Model/InstallModel.php]: Spelling `Fileystem`

File: administrator/components/com_installer/Model/ManageModel.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_installer/Model/ManageModel.php:224]: to what - i am confused

**Comment by brianteeman** [comment]
Conflicts resoolved

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
yay

**Pull request merged**